nautobot/project-static/js/forms.js

```diff
@@ -663,13 +690,36 @@ function initializeDynamicFilterForm(context){
         // 2) combine the two forms into a single set of data without duplicate entries
         let search_query = new URLSearchParams();
         let dynamic_query = new URLSearchParams(new FormData(document.getElementById("dynamic-filter-form")));
+        const urlParams = new URLSearchParams(window.location.search);
+        const non_filter_params = ["saved_view", "sort", "per_page", "table_changes_pending", "clear_view"]
+        urlParams.forEach((value, key) => {
+            if (non_filter_params.includes(key)){
+                search_query.append(key, value)
+            }
+        })
         dynamic_query.forEach((value, key) => { if (value != "") { search_query.append(key, value); }});
         // Some list views may lack a default-filter form
         let default_query = new URLSearchParams(new FormData(document.getElementById("default-filter")?.firstElementChild));
         default_query.forEach((value, key) => {
             if (value != "" && !search_query.has(key, value)) { search_query.append(key, value); }
         });
         $("#FilterForm_modal").modal("hide");
+
+        if (search_query.has("saved_view")) {
+            var all_filters_removed = true
+
+            const non_filter_params = ["saved_view", "sort", "per_page", "table_changes_pending", "all_filters_removed", "clear_view"]
```

Comment:
Would be nice to extract this to a "global" constant in this file rather than redeclaring it in four separate places?